src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/ReissueFurtherEvidenceHandler.java

```diff
@@ -93,7 +93,7 @@ private AbstractDocument<? extends AbstractDocumentDetails> getSelectedDocumentI
                                                                                                     ReissueArtifactUi reissueArtifactUi) {
         return Stream.of(caseData.getSscsDocument(), caseData.getSscsWelshDocuments())
             .flatMap(documents -> getStreamIfNonNull(documents))
-            .filter(document -> isDocumentSelectedInUiEqualsToStreamDocument(reissueArtifactUi, document))
+            .filter(document -> document.getValue().getDocumentLink() != null && isDocumentSelectedInUiEqualsToStreamDocument(reissueArtifactUi, document))
```

Comment:
can we add a unit test for this please, or make sure it's covered in existing tests